DataFormats/MuonDetId/src/classes_def.xml

```diff
@@ -38,9 +38,16 @@
   <class name="DTSectCollId" ClassVersion="10">
    <version ClassVersion="10" checksum="1624682691"/>
   </class>
-  <class name="GEMDetId" ClassVersion="11">
+  <class name="GEMDetId" ClassVersion="12">
+   <version ClassVersion="12" checksum="1195033659"/>
    <version ClassVersion="11" checksum="2300896813"/>
    <version ClassVersion="10" checksum="3587583837"/>
+   <ioread sourceClass = "GEMDetId" version="[-11]" targetClass="GEMDetId" source="" target="">
+    <![CDATA[
+      GEMDetId tmp(GEMDetId::v12Form(newObj->rawId()));
+      *newObj=tmp;
```

Comment:
This assumes that the base part (```DetId```) of ```newObj``` is already filled.
is this guaranteed to be the case?
It is safer perhaps to use the old object (```onfile.rawId()```).